glide.yaml

```diff
@@ -26,6 +26,9 @@ import:
   subpackages:
   - assert
   - require
+- package: github.com/golang/mock
```

Comment:
By the looks of it, yes. It doesn't work locally if you remove the `mock` package from system path. 

Looks like just including thriftrw doesn't pull in the mock in the lockfile.

/cc @breerly